docs/concepts.rst

```diff
@@ -1207,6 +1211,66 @@ queue during retries:
             ti.queue = 'retry_queue'
 
 
+Cluster Policies for Custom Task Checks
+-----------------------------
+You may also use Cluster Policies to  apply cluster-wide checks on Airflow
+tasks. You can raise :class:`~airflow.exceptions.AirflowClusterPolicyViolation`
+in a policy or task mutation hook (described below) to prevent a DAG from being
+imported or prevent a task from being executed if the task is not compliant with
+your check.
+
+These checks are intended to help teams using Airflow to protect against common
+beginner errors that may get past a code reviewer, rather than as technical
+security controls.
+
+For example, don't run tasks without airflow owners:
+
+.. code-block:: python
+    def task_must_have_owners(task: BaseOperator):
```

Comment:
Can you use the lliteralinclude directive to copy the code snippet that is in the `tests/` directory?

Reply:
Here is an example of `literalinclude`:

https://github.com/apache/airflow/blob/efe1bdd29b0a02c37f011c1c5783ef295962260d/docs/howto/operator/google/cloud/cloud_storage_transfer_service.rst#templating

Reply:
Hmm this link seems not to be rendering properly.
![image](https://user-images.githubusercontent.com/11599048/89964712-c7981e00-dbff-11ea-97bf-1d52ea232c74.png)

Reply:
These directives are not supported by Github, but Sphinx does support them and we are actively using them.

Reply:
This translates to https://airflow.readthedocs.io/en/latest/howto/operator/google/cloud/gcs.html#templating after Sphinx renders it.

![image](https://user-images.githubusercontent.com/8811558/90008521-62423c80-dc94-11ea-8dd8-9757848844c3.png)